contracts/stellar-interchain-token-service/src/token_manager.rs

```diff
@@ -24,6 +27,14 @@ impl TokenManagerClientExt for TokenManagerClient<'_> {
     }
 
     fn mint(&self, env: &Env, token_address: &Address, recipient: &Address, amount: i128) {
+        let _: Val = self.execute(
+            token_address,
+            &Symbol::new(env, "mint"),
+            &vec![env, recipient.to_val(), amount.into_val(env)],
+        );
+    }
+
+    fn mint_from(&self, env: &Env, token_address: &Address, recipient: &Address, amount: i128) {
```

Comment:
actually, a proper fix would be to transfer the admin for interchain tokens to their token manager from ITS, so that it can call `mint` directly. This way the same interface can be used for both types. This requires a migration since admin transfer needs to be done for all deployed interchain tokens. Could consider it in a follow up PR perhaps